src/backend/database.ts

```diff
@@ -0,0 +1,5 @@
+import mysql from 'mysql2/promise'
+
+export const connection = await mysql.createConnection('mysql://root:secret@localhost/itslearning')
```

Comment:
```suggestion
export const connection = await mysql.createConnection('mysql://root:secret@localhost/serlo')
```
See https://github.com/serlo/api.serlo.org/pull/1665